index.html

```diff
@@ -207,5 +207,53 @@ <h1>About me</h1>
             </ul>
         </div>
     </div>
+    <form action="https://formspree.io/f/mdovrwgl" method="post">
+        <span>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</span>
+        <ul>
+          <li>
+            <input type="text" aria-label="name" name="user_name" maxlength="30" placeholder="Full name" required/>
+          </li>
+          <li>
+            <input type="email" aria-label="mail" name="user_email" placeholder="Email address" required/>
+          </li>
+          <li>
+            <textarea id="msg" aria-label="user_message" maxlength="500" placeholder="Write me something..." required></textarea>
+          </li>
+          <li class="button">
+            <button type="submit">Get in touch</button>
+          </li>
+        </ul>
+    </form>
+    <footer>
+        <div class="bottomNav">
+            <ul>
+                <li>
+                    <a href="https://github.com/ivangonzalez224">
+                        <img src="images/gitFooter.png" alt="">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/iv%C3%A1n-gonzalez-robles-957491275/">
+                        <img src="images/linkFooter.png" alt="">
+                    </a>
+                </li>
+                <li>    
+                    <a href="https://www.angellist.com/">
+                        <img src="images/angelistFooter.png" alt="">
+                    </a>
+                </li>
+                <li>    
+                    <a href="https://twitter.com/ivang2204">
+                        <img src="images/twitterFooter.png" alt="">
+                    </a>
+                </li>
+                <li>    
+                    <a href="https://medium.com/">
+                        <img src="images/mediumFooter.png" alt="">
+                    </a>
```

Comment:
- [ ] [Optional] 👍 Well done on adding the social icons to your footer! However, I noticed that you haven't provided `alt` attributes for your icon images. It's important to include descriptive `alt` attributes to ensure accessibility for users who rely on assistive technologies. 👍 

Please consider updating your code to include the `alt` attributes for each icon image 🙏, providing a brief description or alternative text for the icon. Here's an example:

```html
<img src="icon.png" alt="Facebook Icon" />
```

By adding the `alt` attributes, you make your website more accessible and provide meaningful information to users who may not be able to see the icons. Great job so far, and keep up the good work! 👍